src/graphql/FlowRun/table-task-runs.gql

```diff
@@ -1,6 +1,6 @@
 query TableTaskRuns(
   $flowRunId: uuid
-  $name: String
+  $run_name: String
```

Comment:
I'm not sure I understand what renaming this variable is meant to do though I don't see any issues with it